services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluLaskentaServiceTest.kt

```diff
@@ -282,46 +283,38 @@ internal class TormaystarkasteluLaskentaServiceTest {
     }
 
     @Nested
-    inner class Pyoraliikenneluokittelu {
+    inner class CalculatePyoraliikenneindeksi {
         // The parameter is only used to call mocks
         val geometriat = setOf<Int>()
 
         @Test
         fun `returns 5 when intersect with priority route`() {
-            every { tormaysService.anyIntersectsWithCyclewaysPriority(geometriat) } returns true
+            every { tormaysService.maxIntersectingPyoraliikenneHierarkia(geometriat) } returns 5
 
-            val result = laskentaService.pyoraliikenneluokittelu(geometriat)
+            val result = laskentaService.calculatePyoraliikenneindeksi(geometriat)
 
-            assertThat(result).isEqualTo(5)
-            verify { tormaysService.anyIntersectsWithCyclewaysPriority(geometriat) }
+            assertThat(result).isEqualTo(5f)
+            verifySequence { tormaysService.maxIntersectingPyoraliikenneHierarkia(geometriat) }
         }
 
         @Test
-        fun `returns 4 when intersect with main route, but not priority route`() {
-            every { tormaysService.anyIntersectsWithCyclewaysPriority(geometriat) } returns false
-            every { tormaysService.anyIntersectsWithCyclewaysMain(geometriat) } returns true
+        fun `returns 3 when intersect with main route, but not priority route`() {
```

Comment:
Maybe these test method names need adjusting too? "main route" --> "other route", "priority route" --> "main route or 'bahn'"? Also, should there be a test for `MUU_YHTEYS`(2)?

Reply:
The laskenta service doesn't know about the different classes, so I renamed the tests neutrally. I changed the non-null tests to a parametrized test.